client/src/components/PageLayout/Footer.tsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, Link, Text } from '@chakra-ui/layout';
+import React from 'react';
+import style from '../../styles/Footer.module.css';
+
+export const Footer: React.FC = () => {
+  return (
+    <Flex
+      flexDirection={['column', 'row']}
+      backgroundColor={'gray.10'}
+      className={style.footer}
+    >
+      <Text className={style.text}>
+        Find out how is your information handled here:
+        <Link className={style.link} href="/policy/">
+          Policy
+        </Link>
+      </Text>
+      <Text className={style.text}>
+        Have an issue? you can check out:
+        <Link
+          className={style.link}
+          href="https://www.freecodecamp.org/news/support/"
+        >
+          Support
```

Comment:
```suggestion
        Found a bug? Ask for help on the forum
        <Link
          className={style.link}
          href="https://www.freecodecamp.org/news/support/"
        >
          the forum
```
/news/support is only really relevant to /learn

Reply:
> It looks a little odd when there isn't much content on the page

I have been thinking, since the issue have been opened, about what to add. I have thought of adding GitHub link, link to discord and some worse thoughts.

But this isn't what the original intent is, I wanted to add policy page link before log in, so people can ready our policy before deciding to commit.

Footer is a mean and isn't an issue so it should solve something, not add headache so I am closing this.

> can you stick it to the bottom of the page?

Yes adding minheight to the content, is the simplest way to solve this,  although it may need some tweeking if itsn't working.

---

I am adding your suggestions and closing it, so it's easier to come back to it later